src/battle.rs

```diff
@@ -169,18 +171,18 @@ pub(crate) fn setup_battle(mut commands: Commands,
 
 pub(crate) fn setup_battle_stats(mut commands: Commands, 
 	asset_server: Res<AssetServer>,
-	mut player: Query<&mut Player>,
-	mut monster_query: Query<(&mut MonsterBundle)>) 
+	mut my_monster: Query<&mut MonsterBundle, (With<Fighting>, Without<Enemy>)>,
+	mut enemy: Query<&mut MonsterBundle, With<Enemy>>) 
 {
-	if player.is_empty() {
+	if my_monster.is_empty() {
 		error!("No player found!");
```

Comment:
It may be more descriptive to say, like, "no player monster found", but this is a nitpick